backend/server/src/main/java/com/issuetracker/controller/web/WebIssueController.java

```diff
@@ -57,6 +57,15 @@ public IssueDetailResponse detailIssue(@PathVariable Long issueId) {
         return issueService.findDetailedIssueId(issueId);
     }
 
+    @PutMapping("/{issueId}")
+    @LoginRequired
+    public void updateIssue(HttpServletRequest request, @RequestBody IssueRequest issue, @PathVariable Long issueId) {
+        String userId = (String) request.getAttribute("userId");
+        UserDto loginUser = authService.getUser(userId);
+        
+        issueService.updateIssue(loginUser, issue, issueId);
```

Comment:
```suggestion
    public void updateIssue(@RequestBody IssueRequest issue, @PathVariable Long issueId) {
        issueService.updateIssue(issue, issueId);
```

issue 를 업데이트 하는데 유저 정보가 왜 필요한지 여쭈어봅니다.

Reply:
위에서 답변 드렸습니다!..

Reply:
ee97bbc 수정했습니다!